src/components/PassphraseForm.jsx

```diff
@@ -79,9 +89,38 @@ class PassphraseForm extends Component {
             {t('ProfileView.password.submit_label')}
           </button>
         </div>
-        <a href='#' className={styles['password-reset-link']}>
+
+        {!passphraseNewRequesting && <a
+          href='#'
+          className={styles['password-reset-link']}
+          onclick={() => this.handleForgot()}>
           {t('ProfileView.password.reset_link')}
-        </a>
+        </a>}
+
+        {passphraseNewRequesting &&
+        <span className={styles['passphrase_new_send']}>
```

Comment:
You should call for Loading component instead.
Like this https://github.com/cozy/cozy-settings/blob/master/src/components/DevicesView.jsx#L21
And then, if the default style for the Loading component isn't quite what you want, add an overwriting declaration for `.set-loading` in the `.styl` of the view.